compiler/src/iree/compiler/Codegen/Common/CPU/Passes.h

```diff
@@ -53,6 +53,10 @@ void addCPUBufferizePasses(OpPassManager &funcPassManager);
 std::unique_ptr<OperationPass<>>
 createCPULowerToUKernelsPass(bool skipIntermediateRoundings = true);
 
+/// Pass to decompose batch_mmt4d/pack/etc to fit ukernel requirements.
+std::unique_ptr<InterfacePass<FunctionOpInterface>>
+createCPUUkernelPreparationPass();
```

Comment:
drive-by style nit: prefer pass names that start with verbs (namespacing if you want): e.g. "PrepareCPUUkernels" or "CPUPrepareUkernels"

(ala "HoistXXX" "ConvertXXX" "DecomposeXXX" "MaterializeXXX")

Reply:
SG! Yeah, I'd like to keep `CPU` prefix for consistency.